packages/patternfly-4/react-core/src/layouts/Level/Level.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/layouts/Level/level';
+
+export interface LevelProps extends React.HTMLProps<HTMLDivElement> {
+   /** Adds space between children. Options are sm, md or lg */
+   gutter?: 'sm' | 'md' | 'lg'; 
+   /** additional classes added to the Level layout */
+   className?: string; 
+   /** content rendered inside the Level layout */
+   children?: React.ReactNode; 
+}
+
+export const Level: React.FunctionComponent<LevelProps> = ({
+  gutter = null,
+  className = '',
+  children = null, 
+  ...props
+}: LevelProps) => (
+  <div
+    {...props}
+    className={css(styles.level, gutter && styles.modifiers.gutter, className)}
```

Comment:
needs to use getGutterModifier function

Reply:
Actually looking at this a little more now, it does not seem like there are other gutter sizes we can apply? Just the one 'pf-m-gutter'? Then the gutter prop type does not make sense. Should probably just be a boolean `hasGutter` or something like that